src/sdk/node_adapter.cc

```diff
@@ -313,5 +313,12 @@ bool NodeAdapter::TransformToColumnKey(hybridse::node::ColumnIndexNode* column_i
     return true;
 }
 
+int64_t NodeAdapter::ConvertToMinute(int64_t time_ms) {
+    if (time_ms == 0) {
+        return 1;
+    }
+    return std::max(NodeAdapter::MIN_TIME, time_ms / 60000 + (time_ms % 60000 ? 1 : 0));
```

Comment:
if time_ms > 0, time_ms / 60000 + (time_ms % 60000 ? 1 : 0) can't <=1, no need to use max function.